stakingModel/contracts/mocks/UBISchemeMock.sol

```diff
@@ -6,15 +6,25 @@ contract UBISchemeMock is UBIScheme {
 
     constructor( Avatar _avatar,
         Identity _identity,
+        FirstClaimPool _firstClaimPool,
         uint256 _initialReserve,
         uint256 _periodStart,
         uint256 _periodEnd,
         uint256 _maxInactiveDays
     )
         public
-        UBIScheme(_avatar, _identity, _initialReserve, _periodStart, _periodEnd, _maxInactiveDays) {}
+        UBIScheme(_avatar, _identity, _firstClaimPool, _initialReserve, _periodStart, _periodEnd, _maxInactiveDays) {}
 
     function start() public {
         isActive = true;
     }
+
+    function end(Avatar _avatar) public onlyAvatar {
```

Comment:
in all the mocks add comment explaining why it was needed

Reply:
updated
3342c9c0a5e4ba58dae711a4431d0da78fc5094e